### PR TITLE
Correct Handling of Edition in Linking of Issues

### LINKED ISSUE #494: Batch page does not correctly identify edition number in URL
The batch page at `/batch/<batchname>/` does not correctly generate the link to the issue(s) if there is an edition that is not a first edition. 

For example, a second edition on a particular date would identify an editionOrder="2" and the xml file identified for the issue would customarily be named `YYYYMMDDEE` where EE is the edition. This should lead to the URL being generated on this page of 
`http://hostname/lccn/<TITLE-LCCN>/<YYYY>-<MM>-<DD>/ed-<E>/`
However, this page is generating ed-1 always.

The regular flow for identifying the edition number and including it as part of the link seems to work just fine in all other locations tested.

### PR DESCRIPTION
**Pull request by BusinessFawn**
- List changes with brief descriptions
  - Closes #494 
@jduss4 @techgique
Adds correct handling of edition in linking of issues in /core/templates/reports/batch.html and /core/templates/issues.html
Unremarkable changes so no update to  README/CHANGELOG

## Submitter Checklist

- [X] Verify all tests pass
  - Run tests with `docker-compose -f test-compose.yml -p onitest up test`
- [x] Update `README.md` and `docs/` as necessary
  - [x] If a `manage.py` command is added, deleted, or modified its help text must
    be valid and it should be documented in detail in
    `docs/advanced/admin-commands.md`
- Update `CHANGELOG.md`
  - [x] Describe change(s) in appropriate section(s)
  - [x] List self in Contributors section
- If a release PR:
  - [x] In `CHANGELOG.md`, replace `[Unreleased]` with version and update compare link
  - [x] Update `core/version.py` with new version
- [X] Resolve merge conflicts
- [X] @mention individual(s) you would like to review the PR
  - Reviews for releases must come from a reviewer at another institution

## Reviewer Checklist

- [ ] Verify all tests pass
- [ ] Review changes for behavior, bugs, and compliance with [Development
  Standards](https://github.com/open-oni/open-oni/tree/dev/CONTRIBUTING.md#development-standards)
  - Request the submitter make any changes rather than pushing changes yourself.
    Otherwise ask another reviewer to look at any changes you have made.
- [ ] If a release, follow the [Release
  Checklist](https://github.com/open-oni/open-oni/tree/dev/CONTRIBUTING.md#release-checklist)

### CODE REVIEW

**Comment by BusinessFawn** [comment]
Seems like there is an problem rendering some data.. Pausing for the moment.

**Comment by BusinessFawn** [comment]
Updated after some better testing. 

Showing my work:

Using [batch_mnhi_german_ver01](https://github.com/open-oni/sample-data/blob/master/batch_mnhi_german_ver01/data/sn83045350/00295876384/1878010301/1878010301_1.xml) updated to edition 101101.

```
<mods:detail type="edition">
  <mods:number>101101</mods:number>
</mods:detail>
```

Before: 

Screen Shot 2020-10-24 at 7 01 59 PM
Which leads to (notice the link points to ed-1)
Screen Shot 2020-10-24 at 7 02 15 PM

After:

Screen Shot 2020-10-24 at 7 05 30 PM
Screen Shot 2020-10-24 at 7 05 44 PM

File: core/templates/issue_pages.html [hunk 1]
```diff
@@ -76,7 +76,7 @@
                           Issue Notes:
                           {{page_info.issue.noteAboutReproduction.text}}.
                         {% endif %}
-                        {% url 'openoni_issue_pages' title.lccn page_info.issue.date_issued issue.edition as url %}
+                        {% url 'openoni_issue_pages' title.lccn page_info.issue.date_issued 1 as url %}
```
Comment (techgique) [inline core/templates/issue_pages.html]: I think you were on the right track here. From looking at the code for the URL a few lines above here, I think the appropriate fix is to replace `1` with `page_info.issue.edition`.

**Review by techgique — CHANGES_REQUESTED** [review]
Please make the suggested change to `core/templates/issue_pages.html`, unless I'm mistaken of course.

Also, a changelog entry for this would be preferred. Something like
```
### Fixed
- URLs/objects built with a static number `1` rather than edition number
```

And then add your name to the `Contributors` list :)

**Review by techgique — APPROVED** [review]

**Pull request merged**